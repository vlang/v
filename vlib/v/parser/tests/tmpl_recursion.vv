fn test_tmpl_catch_recursion() {
	recurse := $tmpl('tmpl/recursive_fail.html')
}