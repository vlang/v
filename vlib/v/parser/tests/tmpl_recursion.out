vlib/v/parser/tests/tmpl/recursive_fail2.html:2:1: error: A recursive call is being made on template recursive_fail
    1 | <p>This makes a recursive call</p>
    2 | @include 'recursive_fail'
      | ~~~~~~~~~

